### DISCUSSION TITLE
Does marqo supports rerankers for text?

### DISCUSSION THREAD

**Discussion by renuka010** [marqo-ai/marqo#883] [Q&A] [answered]
In documentation it was present only for images. Do we have support for text as well?

**Comment by pandu-k** [comment, marked as answer]
Hi @renuka010 . We are looking to support text rerankers soon. We're aiming to get it Q3 this calendar year 👍 

@farshidz For visibility

**Comment by pandu-k** [comment]
On that note, @renuka010 do you mind sharing a bit about your use case? This can help us to steer this feature towards use cases like yours.

**Reply by renuka010** [reply]
Hi @pandu-k . We are already using Marqo in a RAG application for text. We noticed, sometimes quality of documents retrieved are low as documents with good relevancy are ranked lower. It would be great if Marqo had support for reranker, to rearrange the documents according to relevancy.

**Reply by pandu-k** [reply]
Thank @renuka010  - that is helpful feedback. Noted 👍
